### PR TITLE
Kubernetes Prometheus dashboard layout updates

### PR DESCRIPTION
**Pull request by algchoo**
### Description
This PR updates the layout for the Kubernetes Prometheus dashboards.

### Changes
* [updated k8s cluster dashboard json and screenshot](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/commit/6a915682e06f2b6ba15aff0765166b12fb7b3498)
* [updated k8s overview dashboard json and screenshot](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/commit/8d63c3e98b640dc8b4ec81589f32b61df6d7cfd2)
* [updated k8s pod dashboard json and screenshot](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/commit/38f0bead3b67369a2d0ac7efaf364b689e11d8fa)

#### Infrastructure
k8s-overview-prometheus 01
k8s-overview-prometheus 02
k8s-overview-prometheus 03

#### Cluster
k8s-cluster-prometheus

#### Pod
k8s-pod-prometheus

### Additional KPIs
* n/a

### CODE REVIEW

**Review by johnbryan — CHANGES_REQUESTED** [review]
It looks like these are using template variables ${Cluster},${Location},${Namespace}, but they're not actually defined in `dashboardFilters` - I think they need to be added there.

File: dashboards/kubernetes/k8s-pod-prometheus.json [hunk 1]
```diff
@@ -161,242 +190,251 @@
...
               {
-                "tableDisplayOptions": {},
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "STACKED_AREA",
+                "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "sum by (exported_namespace) (kube_pod_status_scheduled{condition=\"true\"})"
+                  "prometheusQuery": "sum by (exported_namespace) (kube_pod_status_scheduled{condition=\"true\",${Cluster},${Location},${Namespace}})",
```
Comment (johnbryan) [inline dashboards/kubernetes/k8s-pod-prometheus.json:360]: for some reason in my project, this metric does not have a `exported_namespace` label. It should always have `namespace` though I believe. Does it work to group by `namespace` instead?

**Review by johnbryan — CHANGES_REQUESTED** [review]
These look great, thanks! Just one comment on a chart that didn't work in my environment.

File: dashboards/kubernetes/k8s-pod-prometheus.json (hunk 1 above)
Comment (johnbryan) [reply dashboards/kubernetes/k8s-pod-prometheus.json:360]: Asked @huygaa11 about this as he's worked more with kube-state-metrics, it _seems_ we should probably use `namespace`
- found a couple questions online where people point to this being a setting https://github.com/prometheus-community/helm-charts/issues/1368. I don't know how common it is either way, but
- the GKE curated kube-state-metrics package (https://cloud.google.com/kubernetes-engine/docs/how-to/configure-metrics#ksm-package) seems to use `namespace`, and this is likely the most common source of kube-state-metrics going forward.

Comment (algchoo) [reply dashboards/kubernetes/k8s-pod-prometheus.json]: This has been changed to `(namespace)` and should be good to go. Hope you're having a great day!

**Review by johnbryan — APPROVED** [review]

**Pull request merged**